routers/record.go

```diff
@@ -78,5 +75,15 @@ func AfterRecordMessage(ctx *context.Context) {
 	if userId != "" {
 		record.Organization, record.User = util.GetOwnerAndNameFromId(userId)
 	}
-	util.SafeGoroutine(func() { object.AddRecord(record) })
+
+	recordSignup := ctx.Input.Params()["recordSignup"]
```

Comment:
Will it panic if `ctx.Input.Params()["recordSignup"]` doesn't exist

Reply:
if ctx.Input.Params()["recordSignup"] doesn't exist it will return ""